sql/views.sql

```diff
@@ -259,5 +259,121 @@ CREATE OR REPLACE VIEW timescaledb_information.data_node AS
     LEFT OUTER JOIN LATERAL @extschema@.data_node_hypertable_info(CASE WHEN s.node_up THEN s.node_name ELSE NULL END) size ON TRUE
   GROUP BY s.node_name, s.node_up, s.owner, s.options;
 
+-- chunks metadata view, shows information about the primary dimension column
+CREATE OR REPLACE VIEW  timescaledb_information.chunks
+AS
+SELECT 
+       hypertable_schema, hypertable_name,
+       schema_name as chunk_schema , chunk_name ,
+       primary_dimension, primary_dimension_type,
+       range_start, range_end,
+       integer_range_start as range_start_integer,
+       integer_range_end as range_end_integer,
+       is_compressed,
+       chunk_table_space as chunk_tablespace,
+       node_list as data_nodes
+       from
+(
+SELECT
+    ht.schema_name as hypertable_schema,
+    ht.table_name as hypertable_name,
+    srcch.schema_name as schema_name,
+    srcch.table_name as chunk_name,
+    dim.column_name as primary_dimension,
+    dim.column_type as primary_dimension_type,
+    row_number() over(partition by chcons.chunk_id order by chcons.dimension_slice_id) as chunk_dimension_num,
+    CASE
+        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
+               dim.column_type = 'TIMESTAMPTZ'::regtype OR
+               dim.column_type = 'DATE'::regtype )
+        THEN _timescaledb_internal.to_timestamp(dimsl.range_start)
+        ELSE NULL
+    END as range_start,
+    CASE
+         WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
+                dim.column_type = 'TIMESTAMPTZ'::regtype OR
+                dim.column_type = 'DATE'::regtype )
+         THEN _timescaledb_internal.to_timestamp(dimsl.range_end)
+         ELSE NULL
+    END as range_end,
+    CASE
+        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
+               dim.column_type = 'TIMESTAMPTZ'::regtype OR
+               dim.column_type = 'DATE'::regtype )
+        THEN NULL
+        ELSE dimsl.range_start
+    END as integer_range_start,
+    CASE
+        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
+               dim.column_type = 'TIMESTAMPTZ'::regtype OR
+               dim.column_type = 'DATE'::regtype )
+        THEN NULL
+        ELSE dimsl.range_end
+    END as integer_range_end,
+    CASE WHEN srcch.compressed_chunk_id is not null THEN 'true'
+         ELSE 'false'
+    END as is_compressed,
```

Comment:
Shouldn't we use PostgreSQL idea of true and false here rather than strings?

```suggestion
    srcch.compressed_chunk_id is NOT NULL AS is_compressed,
```